scenes/game_logic/base_character_behavior.gd

```diff
@@ -1,8 +1,7 @@
 # SPDX-FileCopyrightText: The Threadbare Authors
 # SPDX-License-Identifier: MPL-2.0
 @tool
-# TODO: Prepend "abstract" when switching to Godot 4.5:
-class_name BaseCharacterBehavior
+@abstract class_name BaseCharacterBehavior
```

Comment:
The only reason to hold off on this is that currently https://github.com/inkandswitch/patchwork-godot-plugin/ is based on Godot 4.4; currently one can test it with Threadbare because we don't actually use any 4.5-specific features, but this would change that.

Reply:
It is a very important reason, thanks for letting me know.